specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -52,7 +52,7 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/DataSource"
+              "$ref": "#/definitions/SearchIndexerDataSource"
```

Comment:
This would be breaking for any existing 2019-05-06-preview clients. Shouldn't you first create a new preview version, per Azure service guidelines? Alternatively, you could set the `x-ms-class-name` (or whatever it's called) as long as track 1 doesn't generate off the preview version (in case there's a critical servicing event).

Reply:
Those clients are still in preview, so this should be fine.

Didn't know about `x-ms-class-name` -- thanks for pointing that out.